PWGCF/Flow/Tasks/flowAnalysisGF.cxx

```diff
@@ -577,27 +642,62 @@ struct flowAnalysisGF {
       registry.fill(HIST("pt_dcaXY_dcaZ"), track.pt(), track.dcaXY(), track.dcaZ());
+      if (before) {
+        registry.fill(HIST("trackQA/before/CrossedTPCRows_pt"), track.pt(), track.tpcNClsCrossedRows());
+        registry.fill(HIST("trackQA/before/NumberSharedClustersTPC_pt"), track.pt(), track.tpcNClsShared());
+        registry.fill(HIST("trackQA/before/ITSClusters_pt"), track.pt(), track.itsNCls());
+        registry.fill(HIST("trackQA/before/DCAxy_pt"), track.pt(), track.dcaXY());
+        registry.fill(HIST("trackQA/before/DCAz_pt"), track.pt(), track.dcaZ());
+        registry.fill(HIST("trackQA/before/tpcSignal_pt"), track.pt(), track.tpcSignal());
+        registry.fill(HIST("trackQA/before/phi_pt"), track.pt(), track.phi());
+        registry.fill(HIST("trackQA/before/phi_eta"), track.eta(), track.phi());
+        registry.fill(HIST("trackQA/before/tpcCrossedRowsOverFindableCls_pt"), track.pt(), track.tpcCrossedRowsOverFindableCls());
+      } else {
+        registry.fill(HIST("trackQA/after/CrossedTPCRows_pt"), track.pt(), track.tpcNClsCrossedRows());
+        registry.fill(HIST("trackQA/after/NumberSharedClustersTPC_pt"), track.pt(), track.tpcNClsShared());
+        registry.fill(HIST("trackQA/after/ITSClusters_pt"), track.pt(), track.itsNCls());
+        registry.fill(HIST("trackQA/after/DCAxy_pt"), track.pt(), track.dcaXY());
+        registry.fill(HIST("trackQA/after/DCAz_pt"), track.pt(), track.dcaZ());
+        registry.fill(HIST("trackQA/after/tpcSignal_pt"), track.pt(), track.tpcSignal());
+        registry.fill(HIST("trackQA/after/phi_pt"), track.pt(), track.phi());
+        registry.fill(HIST("trackQA/after/phi_eta"), track.eta(), track.phi());
+        registry.fill(HIST("trackQA/after/tpcCrossedRowsOverFindableCls_pt"), track.pt(), track.tpcCrossedRowsOverFindableCls());
+      }
     }
   }
 
   template <typename CollisionObject, typename TracksObject>
-  inline void FillEventQA(CollisionObject collision, TracksObject tracks)
+  inline void FillEventQA(CollisionObject collision, TracksObject tracks, bool before)
   {
-    registry.fill(HIST("globalTracks_centT0C"), collision.centFT0C(), tracks.size());
-    registry.fill(HIST("PVTracks_centT0C"), collision.centFT0C(), collision.multNTracksPV());
-    registry.fill(HIST("globalTracks_PVTracks"), collision.multNTracksPV(), tracks.size());
-    registry.fill(HIST("globalTracks_multT0A"), collision.multFT0A(), tracks.size());
-    registry.fill(HIST("globalTracks_multV0A"), collision.multFV0A(), tracks.size());
-    registry.fill(HIST("multV0A_multT0A"), collision.multFT0A(), collision.multFV0A());
-    registry.fill(HIST("multT0C_centT0C"), collision.centFT0C(), collision.multFT0C());
+    if constexpr (framework::has_type_v<aod::cent::CentFT0C, typename CollisionObject::all_columns>) {
+      if (before) {
+        // Dit maar gewoon voor en na event selection en dan verschillende wagons met configuration
```

Comment:
Please, for the next iteration, comments only in English. There could be other people learning from your code